puppet/zulip_ops/files/supervisor/conf.d/node_exporter.conf

```diff
@@ -0,0 +1,8 @@
+[program:node_exporter]
```

Comment:
Maybe worth a comment at the top of this mentioning prometheus as well, or including that token in the job name.